### PR TITLE
Fix issue #2287

### PR DESCRIPTION
**Pull request by 733737**
I have tested this and it does work, this will fix bug #2287, it took around 20 minutes of finnicking with code. I am new to programming, so let me know if this is good. It seems to work fine from my test of it...

### CODE REVIEW

File: src/openrct2/audio/Audio.cpp [hunk 1]
```diff
@@ -93,7 +94,18 @@ namespace OpenRCT2::Audio
                 }
             }
         }
-        LoadAudioObjects();
+        while (bool onPrepareDraw = true)
```
Comment (AaronVanGeffen) [inline src/openrct2/audio/Audio.cpp:97]: This is always true?

File: src/openrct2/audio/Audio.cpp [hunk 2]
```diff
@@ -93,7 +94,18 @@ namespace OpenRCT2::Audio
...
+        {
+            if (bool onDraw = false)
```
Comment (AaronVanGeffen) [inline src/openrct2/audio/Audio.cpp:99]: This is always false?

File: src/openrct2/audio/Audio.cpp [hunk 3]
```diff
@@ -93,7 +94,18 @@ namespace OpenRCT2::Audio
...
+            {
+                continue;
+            }
+            else
+            {
+                LoadAudioObjects();
+                break;
```
Comment (AaronVanGeffen) [inline src/openrct2/audio/Audio.cpp:106]: So this will always happen

Comment (733737) [reply src/openrct2/audio/Audio.cpp:106]: yeah, I figured out the code did nothing 5 minutes ago

**Pull request closed without merging**